frontend/src/views/PrivacyPolicy.vue

```diff
@@ -0,0 +1,156 @@
+    </ul>
+    </p>
+    <p>
+    <h2>Contact Details</h2>
+    <div style="width: 100%; overflow: hidden;">
+        <div style="width: 600px; float: left;">
+          <h3>Host</h3>
+          <p>Name: {{ privacyPolicyDetails.HOST_NAME }}</p>
+          <p>Address: {{ privacyPolicyDetails.HOST_ADDRESS }}</p>
+          <p>Contact: <span v-html="privacyPolicyDetails.HOST_CONTACT"></span></p>
+        </div>
+        <div style="margin-left: 620px;">
+          <h3>Administrator</h3>
+          <p>Name: {{ privacyPolicyDetails.ADMIN_NAME }}</p>
+          <p>Address: {{ privacyPolicyDetails.ADMIN_ADDRESS }}</p>
+          <p>Contact: <span v-html="privacyPolicyDetails.ADMIN_CONTACT"></span></p>
+        </div>
+    </div>
+    </p>
+    <p>
+      <h2>The type of personal information we collect</h2>
+      We collect the following information:
+      <ul>
+      <li><strong>User Credentials</strong>: used for administrating user accounts: Email address, username, password. Passwords are not stored or accessible as plain text.</li>
+      <li><strong>Document and Annotations</strong>: Documents are the text files uploaded by project managers and annotations are the labels given to them during the annotation process by annotators. Project managers are responsible for both of these types of data and administrating what is stored. Since annotations may contain free text input for annotators to fill in, it is the responsibility of project managers to warn annotators against entering any personal information into fields like this. Project managers are also responsible for meeting the requirements of the rights of users, explained below in “Your data protection rights”.</li>
+      </ul>
+    </p>
+
+    <p>
+      <h2>How and why {{ productName }} collects personal information</h2>
+      Under the UK General Data Protection Regulation (UK GDPR), the lawful bases we rely on for processing this information are:
+    </p>
+
+    <p>
+      <h3>To carry out our contract with you</h3>
+      When you register for a {{ productName }} account, {{ productName }} collects your username, email address and password which are used to: 
+      allow you to log in and use {{ productName }} securely; verify your identity before using the service; contact you if required regarding 
+      your data, account or updates to the terms of service.
+      
+      When project managers and/or administrators upload documents for annotation and annotations are made, 
+      data is stored for the purposes of managing annotation tasks and providing {{ productName }}’s functionality.
+    </p>
+
+    <p>
+      <h3>When we believe it is in our legitimate interests</h3>
+      When you use {{ productName }} the host stores your IP address in the form of request logs. Logs are not shared.
```

Comment:
This line needs a "why" as well as a "what", e.g.

> the host stores your IP address in the form of request logs, which are used to monitor and diagnose any issues with the web service.  Such logs are not shared except if necessary to comply with a legal obligation, and are deleted once they are no longer required, typically within a few weeks of their creation.

Don't know if we need to be any more specific than "a few weeks".

Reply:
>Don't know if we need to be any more specific than "a few weeks".

How long are the logs kept?

Reply:
In our specific case for the production deployments in Sheffield it's roughly 14 days but this is supposed to be a generic template so maybe we need some sort of placeholder for `logRetention` and use a vague statement if that config option is not set.

Reply:
Maybe we should just go with the vague option of
> typically within a few weeks of their creation.